src/composition.cc

```diff
@@ -121,7 +125,7 @@ Composition::Ptr Composition::NewDecay(int delta) {
   if (atom_.size() == 0)
     return decayed;
 
-  decayed->atom_ = pyne::decayers::decay(atom_, 2419200.0 * delta);
+  decayed->atom_ = pyne::decayers::decay(atom_, static_cast<double>(secs_per_timestep) * delta);
```

Comment:
No - this cast is different - it will still be necessary.  pyne::decayers::decay takes a double for the time argument - so I casted secs_per_timestep in order to promote the expression value to a double.